### ISSUE TITLE
No "dist" folder on a repo

### ISSUE THREAD

**Issue by aik099** [bitovi/syn#111]
The README says that in `dist` folder there is a standalone JS file, but there is no such folder in the repo.

**Comment by SergioCrisostomo** [comment]
@aik099 the dist folder only exists in the tagged release. Take a look [here](https://github.com/bitovi/syn/tree/v0.1.4). 
So it's tagged and then the dist folder is removed until next release.

**Comment by aik099** [comment]
But README says click here to download package and `master.zip` is downloaded which does't have it.

I guess that part of README needs to be updated to:

* remove the link
* mention that `dist` folder only exists on releases

**Comment by PhiLhoSoft** [comment]
The dist folder is also missing in the npm release, which breaks the test scripts. It was there is previous versions.

**package.json**:
```
  "name": "syn",
  "version": "0.1.4",
  "main": "dist/cjs/syn.js",
```
Was not in `dist/cjs`, was directly in `dist`.
**bower.json**:  
```
  "name": "syn",
  "version": "0.0.3",
```
(They are not in sync!)

Can you fix your build / deploy process, please? Thanks in advance.

**Comment by daffl** [comment]
I released version 0.1.5 with the old folder structure and 0.2.0 with the new build and updated the readme with the correct download link. According to [semantic versioning](http://semver.org/) this should not break your app as long as you are referencing `0.1.5` directly or a version range like `^0.1.5`.

I forgot to update the `bower.json` version number but since Bower is completely based off of git tags I don't think it really matters.

**Comment by PhiLhoSoft** [comment]
Not sure why you have closed this issue. I just did a npm install of syn 0.2.0, and all I get are top-level files and a src folder (with JS files, of course). Still no dist folder, unlike what is described in the readme.
So either I am dense and missing something, or there is something messed up.
